### PR TITLE
Initial nori port

### PR DESCRIPTION
**Pull request by zvecr**
Port of [nori](http://www.40percent.club/2018/10/nori.html) kbfirmware.com json file (https://github.com/di0ib/Misc/blob/master/nori/nori.json) following what has been previously done for other 40percent.club boards.

### CODE REVIEW

**Comment by drashna** [comment]
Travis CI errors can be ignored here.  The issues are @wanleg's keymap and this board not supporting the SWAP HANDs feature, and my keymap being too large...

**Comment by zvecr** [comment]
@drashna as a FYI, i stole the default lets_split keymap so a lot of the raised issues will also apply to that keymap too. Would you approve a PR to resolve that?

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        persistent_default_layer_set(1UL<<_COLEMAK);
+      }
+      return false;
+      break;
+    case DVORAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_dvorak);
+        #endif
+        persistent_default_layer_set(1UL<<_DVORAK);
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+    case COLEMAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_colemak);
+        #endif
+        persistent_default_layer_set(1UL<<_COLEMAK);
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_qwerty);
+        #endif
+        persistent_default_layer_set(1UL<<_QWERTY);
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_colemak);
+        #endif
+        persistent_default_layer_set(1UL<<_COLEMAK);
+      }
+      return false;
+      break;
+    case DVORAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+          PLAY_SONG(tone_qwerty);
+        #endif
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+    case COLEMAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+  _______, RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL, \
+  _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+)
+
+
+};
+
+#ifdef AUDIO_ENABLE
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

File: keyboards/40percentclub/nori/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/40percentclub/nori/keymaps/default/keymap.c]: Done

**Comment by drashna** [comment]
Yes, absolutely.

**Comment by drashna** [comment]
Travis CI errors can be ignored (gosh, that drashna guy and his too huge keymaps)

**Comment by drashna** [comment]
Awesome, looks great!

**Pull request merged**